server/proxy-server.js

```diff
@@ -1,14 +1,22 @@
 const express = require('express');
 const parser = require('body-parser');
+const _ =require('underscore');
+const fs = require('file-system');
 
 const app = express();
 
 app.use(parser.json());
 
 app.use(express.static(__dirname  + '/dist'));
-
+app.get('/:id', (req, res) => {
+  fs.readFile(__dirname + '/dist/template.html', 'utf8', function (err, html) {
```

Comment:
not sure if you are using an async version for a specific reason, but could use readFileSync and mitigate the callback. not necessary though at all, this works wonders too